tests/network/vmi_networking.go

```diff
@@ -259,8 +260,18 @@ var _ = SIGDescribe("[rfe_id:694][crit:medium][vendor:cnv-qe@redhat.com][level:c
 			Expect(err).ToNot(HaveOccurred())
 
 			libwait.WaitUntilVMIReady(e1000VMI, console.LoginToAlpine)
-			// as defined in https://vendev.org/pci/ven_8086/
-			checkNetworkVendor(e1000VMI, "0x8086")
+
+			By("verifying vendors for respective PCI devices")
+			const findCmd = `find /sys/devices -type d -name "%s" -exec cat {}/vendor \;`
```

Comment:
Could you please describe in the commit message what this command is supposed to do?
What is the reason it ends with an escaped semi-colon?

Reply:
Since we know the PCI address in advance, WDYT about using the following file
```bash
$ cat /sys/bus/pci/devices/0000:01:00.0/vendor
0x17cb
```
?

I find it more straightforward.